contracts/connection-router/src/contract/execute.rs

```diff
@@ -103,18 +103,38 @@ pub fn unfreeze_chain(
     Ok(Response::new().add_event(ChainFrozen { name: chain }.into()))
 }
 
+pub fn require_admin(deps: &DepsMut, info: MessageInfo) -> Result<(), ContractError> {
+    let config = CONFIG.load(deps.storage)?;
+    if config.admin != info.sender {
+        return Err(ContractError::Unauthorized);
+    }
+    Ok(())
+}
+
+pub fn require_governance(deps: &DepsMut, info: MessageInfo) -> Result<(), ContractError> {
+    let config = CONFIG.load(deps.storage)?;
+    if config.governance != info.sender {
+        return Err(ContractError::Unauthorized);
+    }
+    Ok(())
+}
```

Comment:
take config and sender instead? the contract can already load config

Reply:
I didn't make any change to these 2 functions but moved them up above the contract struct impl. I'd wanna avoid any change to them in this PR.